var/spack/repos/builtin/packages/dftbplus/package.py

```diff
@@ -62,16 +76,38 @@ class Dftbplus(MakefilePackage):
         default=False,
         description="Use DftD3 dispersion library " "(if you need this dispersion model)",
     )
+    variant(
+        "api",
+        default=True,
+        description="Build the API library " "(if you need to link to DFTB+ from other codes)",
```

Comment:
Why is this split into two strings?

Reply:
apparently thats black's magic, I had kept them in two lines, black thought its short enough to be in one line, and you dislike string concat in one line

Reply:
If they fit in one line, is there any reason not just have a single string without any concatenation?

Reply:
Edit: I saw you did this in a latter commit.